### PR TITLE
Update docs with metrics observation instructions

### PR DESCRIPTION
**Pull request by Bslabe123**

### CODE REVIEW

File: docs/observability-prometheus-metrics-in-jetstream-server.md [hunk 1]
```diff
@@ -48,4 +48,38 @@ jetstream_prefill_backlog_size{id="SOME-HOSTNAME-HERE>"} 0.0
...
+          "node"
+        ]
+      }
+    }
+  }
+  ```
+
+The metrics can now be queried in the [Google Cloud Metrics Explorer](https://pantheon.corp.google.com/monitoring/metrics-explorer). When adding a metrics query with the `+Add Query` button the new metrics should be found under the `Prometheus Target > Jetstream` submenu.
+
+Additional guides on the metrics explorer can be found [https://cloud.google.com/monitoring/charts/metrics-selector](here).
```
Comment (JoeZijunZhou) [inline docs/observability-prometheus-metrics-in-jetstream-server.md:85]: this link has some issue.

File: docs/observability-prometheus-metrics-in-jetstream-server.md [hunk 2]
```diff
@@ -48,4 +48,38 @@ jetstream_prefill_backlog_size{id="SOME-HOSTNAME-HERE>"} 0.0
 # HELP jetstream_slots_used_percentage The percentage of decode slots currently being used
 # TYPE jetstream_slots_used_percentage gauge
 jetstream_slots_used_percentage{id="<SOME-HOSTNAME-HERE>",idx="0"} 0.04166666666666663
-```
\ No newline at end of file
+```
+
+## Observe metrics on GKE clusters
+
+The following applies only for Jetstream deployed on a GKE cluster. Currently [Google Cloud Managed Service for Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus) is enabled by default on all GKE clusters, it determines scrape targets via the [PodMonitoring](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/v0.10.0/doc/api.md#podmonitoring) custom resource, consequently one must be applied to your cluster, the following .json manifest can be used as a template:
```
Comment (JoeZijunZhou) [inline docs/observability-prometheus-metrics-in-jetstream-server.md:55]: How does users/customers use this .json manifest? Is this a required step to access metrics on GCP console?

Comment (Bslabe123) [reply docs/observability-prometheus-metrics-in-jetstream-server.md:55]: The podmonitoring is essentially directions to GCP managed prometheus on where to scrape metrics from so its neccesary, more about it [here](https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed#gmp-pod-monitoring)

Comment (JoeZijunZhou) [reply docs/observability-prometheus-metrics-in-jetstream-server.md:55]: Could you add this to the README?

File: docs/observability-prometheus-metrics-in-jetstream-server.md [hunk 3]
```diff
@@ -48,4 +48,38 @@ jetstream_prefill_backlog_size{id="SOME-HOSTNAME-HERE>"} 0.0
 # HELP jetstream_slots_used_percentage The percentage of decode slots currently being used
 # TYPE jetstream_slots_used_percentage gauge
 jetstream_slots_used_percentage{id="<SOME-HOSTNAME-HERE>",idx="0"} 0.04166666666666663
-```
\ No newline at end of file
+```
+
+## Observe metrics on GKE clusters
+
+The following applies only for Jetstream deployed on a GKE cluster. Currently [Google Cloud Managed Service for Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus) is enabled by default on all GKE clusters, it determines scrape targets via the [PodMonitoring](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/v0.10.0/doc/api.md#podmonitoring) custom resource, consequently one must be applied to your cluster as follows:
```
Comment (JoeZijunZhou) [inline docs/observability-prometheus-metrics-in-jetstream-server.md:55]:
```suggestion
The following applies only for Jetstream deployed on a GKE cluster. Currently [Google Cloud Managed Service for Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus) is enabled by default on all GKE clusters, it determines scrape targets via the [PodMonitoring](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/v0.10.0/doc/api.md#podmonitoring) custom resource. After you deployed the JetStream GKE workload, you need to apply the PodMonitoring resource to your cluster as follows:
```

**Review by JoeZijunZhou — APPROVED** [review]

**Pull request merged**